includes/application/handleAppl.php

```diff
@@ -159,7 +159,26 @@
 		
 		$curDate = date('Y-m-d'); 
 		
-		if( ( strtotime($curDate) - strtotime($genDate) ) > 10*24*60*60 )
+		$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
+
+		if ( $sqlconn->connect_errno )
+		{
+			echo "Internal Server Error, Sorry for inconvenience.";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->